sidekick/utils.py

```diff
@@ -11,3 +15,44 @@ def get_current_week_number():
 
 def clean_message(message):
     return " ".join(message.replace("\n", " ").replace("\t", " ").split())
+
+
+def get_whatsapp_contact(org, msisdn):
+    """
+    Returns the WhatsApp ID for the given MSISDN
+    """
+    distribution = pkg_resources.get_distribution("rp-sidekick")
+
+    response = requests.post(
+        urllib_parse.urljoin(org.engage_url, "/v1/contacts"),
+        json={"blocking": "wait", "contacts": [msisdn]},
+        headers={
+            "Authorization": "Bearer %s" % (org.engage_token,),
+            "User-Agent": "rp-sidekick/{}".format(distribution.version),
+        },
+    )
+    response.raise_for_status()
```

Comment:
Nice use of `raise_for_status` -> I'm just wondering about our error handling here as this is a fairly common pattern. If we don't handle the exception, we get an error in Sentry, but what happens to the state of the system when this and other calls fail? I realize that this is an unlikely use case here though, so happy to leave this for now.

Reply:
Made a note about this and will look into this later. Just need to get this out at the moment.